### ISSUE TITLE
Position of focal point is not set when selecting crops

### ISSUE THREAD

**Issue by bjarnef** [Offroadcode/Zoom-Area-Cropper#2]
When you are selecting each crop, it doesn't change position of focal point.
E.g. you might want to adjust an existing crop a bit, but you have to set the focal point again.

![image](https://cloud.githubusercontent.com/assets/2919859/26558333/1f496084-44a8-11e7-9916-cc8076e1d1b8.png)

**Comment by cssquirrel** [comment]
This shouldn't be happening, as I haven't had this issue in my tests. I'll go through again and see if I can find a scenario that causes this. :/

**Comment by cssquirrel** [comment]
@bjarnef Are you using v1.0.0 or v1.0.1? And what's the size of your test image? v1.0.0 had a bug similar to what you reported that was corrected in v1.0.1, so checking.

**Comment by cssquirrel** [comment]
I cannot reproduce this in v1.0.1, I can reproduce this in v1.0.0.  

I'm going to mark this as closed and fixed by v1.0.1 unless I can get a report of failure on the new version.

**Comment by cssquirrel** [comment]
Fixed in [v1.0.1](https://github.com/Offroadcode/Zoom-Area-Cropper/releases/tag/v1.0.1)

**Comment by bjarnef** [comment]
It was v1.0.1, but I can also reproduce it in v1.0.2

I think it happens because of the large image (4454 x 3191px): https://media.koreus.com/201701/allez-faire-loutre.jpg

Sometimes it does position the focal point correct, when reloading the browser window, but not always.

**Comment by cssquirrel** [comment]
I've tested this in both versions using your providing image, and haven't been able to reproduce it.

Hmm, it could be the load time for the image may cause the image size to not be set correctly, which would confuse the code responsible for positioning the handle. My tests are on localhost and the lag of downloading the image is likely the culprit.

This is almost certainly then the same root cause for #3. I've got some ideas on how to handle this.
